oneview_redfish_toolkit/api/blade_manager.py

```diff
@@ -39,9 +39,9 @@ def __init__(self, server_hardware):
 
         super().__init__(server_hardware, server_hardware['mpFirmwareVersion'])
 
-        self.redfish["FirmwareVersion"] = server_hardware['mpFirmwareVersion']
         self.redfish["ManagerType"] = "BMC"
         self.redfish["Name"] = "Blade Manager"
+        self.redfish["Description"] = "HPE Integrated Lights Out"
```

Comment:
This isn't really the iLO that we are exposing in this service.  Shouldn't the description be something more like 'Synergy Blade Manager' or 'HPE OneView Blade Manager'?

Reply:
If the manager type is a BMC (baseboard managmenet controller) -- then for HPE gear, that iLO.  Other vendors have other names (BMC, DRAC, etc.)

https://hewlettpackard.github.io/ilo-rest-api-docs/ilo5/?python#find-the-ilo-5-management-processor
seems to indicate that iLO uses them interchangeably.


Are you saying that the description is for the logical redfish manager, and not the underlying hardware component?